docs/search_relevance/tests/008-BoolBoosted-vs-PhaserBeam.md

```diff
@@ -0,0 +1,54 @@
+# Test 6 - `ConstScore` vs `BoolBoosted`
+
+## Overview
+
+Currently we are matching _named features_<sup>*</sup> with a simple **AND** query.
```

Comment:
Might be nice to link to bellow like this, took me a second to find what this refers to: https://stackoverflow.com/a/48453745

Reply:
https://github.com/wellcomecollection/catalogue/pull/459/commits/9c25387522a1255826ac09dd106f100e2c6cf9de